src/main/java/org/zendesk/client/v2/model/Ticket.java

```diff
@@ -16,8 +16,6 @@ public class Ticket extends Request implements SearchResultEntity {
     private static final long serialVersionUID = 1L;
```

Comment:
Did you test the serialization/deserialization ?
I'm not sure how Java exactly handles variable hiding with serialization, but I have a high doubt this change keeps the compatibility.

Reply:
I did some testing and I don't think the change will break the deserilization but it IS a breaking change:

```
    static class A implements Serializable {
        private static final long serialVersionUID = 1L;
        protected String field;
    }

    static class B extends A implements Serializable {
        private static final long serialVersionUID = 1L;
        // to remove onced we serialized
        private String field;
    }

    public static void main(String[] args) throws IOException, ClassNotFoundException {
        if(args != null && args.length >0) {
            var serializedValue = Base64.getDecoder().decode(args[0]);
            try(var bais = new ByteArrayInputStream(serializedValue)) {
                try(var ois = new ObjectInputStream(bais)) {
                    B b = (B) ois.readObject();
                    System.out.println(b);
                    System.out.println(b.field);
                }
            }
            return;
        }

        var b = new B();
        b.field = "toto";

        try (var baos = new ByteArrayOutputStream()) {
            try (var oos = new ObjectOutputStream(baos)) {
                oos.writeObject(b);
                System.out.println(Base64.getEncoder().encodeToString(baos.toByteArray()));
            }
        }
    }

}
```

Running this once to get a serialized version of `B`, then removing the private hiding field (like this PR does), I can confirm the deserialization process still works but the value of `field` is now null.

Reply:
```suggestion
    private static final long serialVersionUID = -7559199410302237012L;
```